lib/generators/hyrax/templates/app/controllers/curation_concerns/admin_controller.rb

```diff
@@ -1,4 +1,4 @@
-module CurationConcerns
+module Hyrax
```

Comment:
I'm not sure we need this file. If we do, it should not have `curation_concerns` in its path.